studentvue/tests/test_course_history.py

```diff
@@ -0,0 +1,17 @@
+from unittest import TestCase
```

Comment:
This is fine for now, but my plan for tests is to abstract the parsing of the html / data from the fetching of the data, hence the static `_parse...` methods. This way, there are no credentials involved, and you can just pass in some mocked up html and see if it spits out the right result, so you can also test edge cases beyond what's in your own account.